src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstiertodeliusupdate/listeners/HMPPSTierListener.kt

```diff
@@ -38,19 +39,23 @@ class HMPPSTierListener(
           telemetryService.successfulWrite(crn, calculationId)
         }
       }
+    } catch (e: WebClientResponseException.NotFound) {
+      if (e.responseBodyAsString.contains("Offender with CRN %s not found".format(crn))) {
```

Comment:
could you use interpolation rather than a String.format here instead?
```suggestion
      if (e.responseBodyAsString.contains("Offender with CRN $crn not found")) {
```